src/browser/bounds_changed_state_tracker.js

```diff
@@ -291,7 +290,7 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
                 }
 
                 groupLeader = WindowGroupTransactionTracker.getGroupLeader(groupUuid) || {};
-                if (groupLeader.name === name && checkTrackingApi(groupLeader)) {
+                if (groupLeader.name === name && checkTrackingApi(groupLeader) && !ofWindow.skipNext) {
                     var delta = getBoundsDelta(currentBounds, cachedBounds);
```

Comment:
hacky way to get around the infinite loop that occurred with programmatic moves....